src/marqo/core/models/marqo_add_documents_response.py

```diff
@@ -31,14 +18,47 @@ def get_header_dict(self) -> Dict[str, str]:
         }
 
 
-class MarqoAddDocumentsResponse(MarqoBaseModel):
+class MarqoBaseDocumentsResponse(MarqoBaseModel):
+    """A base documents API response model."""
+    _batch_response_stats: BatchResponseStats = Field(exclude=True, default_factory=BatchResponseStats)
+
+    def dict(self, *args, **kwargs) -> Dict[str, Any]:
+        """Setting default exclude to exclude _batch_response_stats from the response.
+
+        Setting Field(exclude=True) does not work for
```

Comment:
Is this a known pydantic issue?

Reply:
We are using pydantic 1 so I can't find anything about this. I did a test on Pydantic 2 and the bug is resolved